### ISSUE TITLE
[3.7.1 regression] androidx.annotation.Nullable is not a type annotation

### ISSUE THREAD

**Issue by krocard** [typetools/checker-framework#3935]
``` java
import androidx.annotation.Nullable;
class Fuu {
    @Nullable private byte[] data;
}
```
 fails in 3.7.1 (but passes in 3.7.0), with the following error:
```
error: [nullness.on.primitive] nullness annotations are not applicable to primitive types
    @Nullable private byte[] data; 
                      ^
```

This suggests that the nullness checker considers Androidx's `@Nullable` a type annotation like `@NullableType`, when it is in fact a field one.
Aka `byte @Nullable [] data` is not legal.

**Comment by krocard** [comment]
Thanks for the quick fix!

**Comment by mernst** [comment]
FYI, there will be a new Checker Framework release today, since it's the first business day of the month.
